src/test/java/uk/nhs/prm/deduction/e2e/queue/QueueMessageHelper.java

```diff
@@ -29,7 +30,7 @@ public void deleteAllMessages() {
         log(String.format("Trying to delete all the messages on : %s", this.queueUri));
         try {
             thinlyWrappedSqsClient.deleteAllMessages(queueUri);
-        } catch (Exception e){
+        } catch (Exception e) {
```

Comment:
Can we find what exception is being thrown here and be explicit, such as `IllegalArgumentException` :) Additionally, please rename `e` in favour of `exception`. 

```suggestion
        } catch (Exception exception) {
```